build.gradle

```
@@ -1,7 +1,13 @@
+buildscript {
+    ext {
+        springBootVersion = '3.0.13'
+    }
+}
+
 plugins {
     id 'java'
     id 'io.freefair.lombok' version '8.10.2'
-    id 'org.springframework.boot' version '2.7.18'
+    id 'org.springframework.boot' version "${springBootVersion}"
```

I don't know if I really like the proposal to use a property in `buildscript` block.
It had several lines without being an elegant solution for future improvements.
I think it would remain more straightforward to keep this
```suggestion
    id 'org.springframework.boot' version '3.0.13'
```

---

Resolved in [52286dc](https://github.com/iExecBlockchainComputing/iexec-sms/pull/285/commits/52286dc3d3a52e6b40854025aa76db758489a310)